internal/core/src/futures/future_c.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2019-2020 Zilliz. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under the License
+
+#include <memory>
+
+#include "future_c.h"
+#include "folly/init/Init.h"
+#include "Future.h"
+
+extern "C" CFollyInit*
+folly_init() {
+    auto opts = folly::InitOptions{};
+    opts.useGFlags(false);
+    auto initializer = new folly::Init(nullptr, nullptr, opts);
+    return static_cast<CFollyInit*>(static_cast<void*>(initializer));
```

Comment:
add a config to contorl fully pool size.
by default is cpu core, and we can set it to cpu core * 2?

and search and query may already have a ppol?

Reply:
It just work for UT now, I will add real pool support in next PR which use async cgo in search/query.